### ISSUE TITLE
Addition/Clarification to the DPM installation documentation

### ISSUE THREAD

**Issue by LeonLaude** [MicrosoftDocs/feedback#2958]
**Is your feature request related to a problem? Please describe.**
There are many questions coming in if System Center Data Protection Manager (DPM) is supported on a VMware virtual machine, the answer is yes and it is documented, but it should be documented in the main installation documentation page.

**Describe the solution you'd like**
I would like the [Get DPM installed](https://docs.microsoft.com/en-us/system-center/dpm/install-dpm?view=sc-dpm-2019) page which is the _main product installation page_ to have the information of support for VMware virtual machines.

**Additional context**
Currently, this information can be found on the [What's supported and what isn't for DPM?](https://docs.microsoft.com/en-us/system-center/dpm/dpm-support-issues?view=sc-dpm-2019) page, but this page is pretty hidden, I want it to be clearer for everyone to understand where a DPM server can be installed on.

Basically the text below should be moved or added to the "Get DPM installed" page:

_"As a Windows virtual machine in VMWare - From DPM 2012 R2 Update 5 onwards you can install DPM on a Windows virtual machine in a VMWare environment. In this configuration DPM can protect Microsoft workloads that are all running as Windows virtual machines in VMWare."_

Best regards,
Leon

**Comment by JasonWHowell** [comment]
@john-par please take a look at this doc enhancement suggestion.

**Comment by JasonWHowell** [comment]
@rayne-wiselman  please take a look at this doc enhancement suggestion. Thanks! Jason

**Comment by rayne-wiselman** [comment]
please assign to @JYOTHIRMAISURI

**Comment by JasonWHowell** [comment]
#reassign:JYOTHIRMAISURI

**Comment by jyothisuri** [comment]
Thanks for your dedication to our documentation. Unfortunately, at this time we have been unable to review your issue in a timely manner and we sincerely apologize for the delayed response. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. 

#please-close
